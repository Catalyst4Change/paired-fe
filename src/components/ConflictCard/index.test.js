import React from 'react';
import { shallow } from 'enzyme';
import { ConflictCard } from './index';

describe('ConflictCard', () => {
  const mockDate = 'Wed Apr 17 2019';
  const mockTime = 'morning';
  const mockHistory = { push: jest.fn() };
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <ConflictCard date={mockDate} time={mockTime} history={mockHistory} />
    );
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
