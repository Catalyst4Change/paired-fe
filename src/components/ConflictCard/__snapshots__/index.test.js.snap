// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConflictCard should match the snapshot 1`] = `
<div>
  <h3>
    Sorry you have a conflict
  </h3>
  <p>
    You already have a pairing scheduled for 
    Wed Apr 17 2019
     from 
    8:00 - 8:50 a.m.
  </p>
  <p>
    Please go back and choose a different date/time or adjust your schedule
  </p>
  <button
    className="ConflictCard--button"
    onClick={[Function]}
  >
    Choose Another Pairing
  </button>
  <button
    className="ConflictCard--button--schedule"
    onClick={[Function]}
  >
    View Schedule
  </button>
</div>
`;
